filebeat/input/docker/config.go

```diff
@@ -40,9 +41,12 @@ type config struct {
 }
 
 type containers struct {
+	// IDs + Path to be removed in 8.0 in favor of Paths.
```

Comment:
I think it can be still helpful to keep the ids field long-term for docker use cases, specially if we implement in the future another input for different runtimes.
Do you think that would be any problem with keeping both options?

Reply:
not a problem. removing this comment.